### PR TITLE
color picker input completed

### PR DESCRIPTION
**Pull request by Ddupasquier**
@nccandiotti 
Here's another component to review!
Just a color picker component. Nothing too crazy.

### CODE REVIEW

File: src/lib/inputs/InputColor.svelte [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+    // Event dispatcher
+    const dispatch = createEventDispatcher();
+
+    // Variables
+    let inputClassList = ["input", size];
+    let inputClassString = "";
+
+    // Lifecycle Hooks
+    onMount(() => {
```
Comment (nccandiotti) [inline src/lib/inputs/InputColor.svelte:21]: are you generally using double quotes here?

File: src/lib/inputs/InputColor.svelte [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    });
+
+    // Reactive Statements
+    $: {
+        inputClassList = ["input", size];
+        inputClassString = inputClassList.join(" ");
+    }
+    $: inputStyle = `
+        ${style}
+    `;
```
Comment (nccandiotti) [inline src/lib/inputs/InputColor.svelte]: could prob be on 1 line no?

File: src/lib/inputs/InputColor.svelte [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+        &::-moz-focus-inner {
+            border: 0;
+            padding: 0;
+            outline: none;
+        }
+    }
+
+    .label {
+        font-size: 0.8rem;
```
Comment (nccandiotti) [inline src/lib/inputs/InputColor.svelte]: would consider 16px or exposing it as a scss var for a11y

File: src/lib/inputs/InputColor.svelte [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        background: transparent;
+    }
+
+    input[type="color"] {
+        -webkit-appearance: none;
+        -moz-appearance: none;
+        appearance: none;
+        background-color: transparent;
+        border: none;
+        cursor: pointer;
```
Comment (nccandiotti) [inline src/lib/inputs/InputColor.svelte:80]: in general for inputs are you changing styling for when they're disabled, like graying out background or cursor: default or cursor: not-allowed

File: src/lib/inputs/InputColor.svelte (hunk 1 above)
Comment (Ddupasquier) [reply src/lib/inputs/InputColor.svelte:21]: I use whatever my formatter prefers.

File: src/lib/inputs/InputColor.svelte (hunk 2 above)
Comment (Ddupasquier) [reply src/lib/inputs/InputColor.svelte]: Not important.

File: src/lib/inputs/InputColor.svelte (hunk 3 above)
Comment (Ddupasquier) [reply src/lib/inputs/InputColor.svelte]: Interesting. Could you elaborate? I'm not familiar with this concept.

File: src/lib/inputs/InputColor.svelte (hunk 4 above)
Comment (Ddupasquier) [reply src/lib/inputs/InputColor.svelte:80]: The background will become #ccc and the disabled attribute will be added to the element.

Comment (Ddupasquier) [reply src/lib/inputs/InputColor.svelte:80]: Though, on review, I'm seeing that I didn't do that with this component... xD Fixing now.

Comment (Ddupasquier) [reply src/lib/inputs/InputColor.svelte:80]: For this use case, a filter on the component will suffice.

**Pull request merged**